src/applications/hca/components/IDForm.jsx

```diff
@@ -75,6 +75,47 @@ export default class IDForm extends React.Component {
     },
   };
 
+  renderServerErrorAlert = () => {
```

Comment:
These are just placeholders at the moment. Will get further direction from UX/copy on what the content of these error alerts should be.